.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,7 +1,7 @@
 contact_links:
-  - name: Ask a question during our office hours
-    url: https://calendly.com/fivetran-solutions-team/fivetran-solutions-team-office-hours
-    about: Schedule time during the external office hours block with the Fivetran Analytics Engineering team for support
+  - name: Ask a question during our office hours 
+    url: https://calendly.com/fivetran-solutions-team/fivetran-solutions-team-office-hours 
+    about: Schedule time during the external office hours block with the Fivetran Analytics Engineering team for support 
```

Comment:
oh interesting I copied/pasted from another package. Did we bulk update this across packages?

Reply:
We are doing a bit of A/B testing to see the impact and benefit of continue to point folks to office hours vs pointing to the community. For new packages we should point to the community. I imagine we will want to scale out to the res of the packages at some point.

I don't have any copy for this specific community point, it would be great if you could come up with one and we can review and look to leverage in other package.

Reply:
This is great, thanks @fivetran-avinash!